component/core/src/main/java/org/exoplatform/social/core/manager/ActivityManagerImpl.java

```diff
@@ -402,6 +402,9 @@ public void deleteComment(ExoSocialActivity existingActivity, ExoSocialActivity
    * {@inheritDoc}
    */
   public void saveLike(ExoSocialActivity existingActivity, Identity identity) {
+    existingActivity.setTitle(null);
+    existingActivity.setBody(null);
+    existingActivity.setTemplateParams(null);
```

Comment:
This workarounds the problem, could we know why they are encoded and prevent it.

Reply:
Okay i will check